opm/simulators/timestepping/AdaptiveTimeStepping.hpp

```diff
@@ -97,6 +97,16 @@ namespace Opm {
 
         void setSuggestedNextStep(const double x) { suggested_next_timestep_ = x; }
 
+        void updateTUNING(const Tuning& tuning, size_t time_step) {
+            restart_factor_ = tuning.getTSFCNV(time_step);
+            growth_factor_ = tuning.getTFDIFF(time_step);
+            max_growth_ = tuning.getTSFMAX(time_step);
+            max_time_step_ = tuning.getTSMAXZ(time_step);
+            suggested_next_timestep_ = tuning.getTSINIT(time_step);
```

Comment:
I think this had an unwanted side effect: if there is no change in TUNING, we will still set the next timestep to be the (typically small) timestep from TUNING. So every report step must start with 0.1 day for example. To handle this we must know if TUNING was given *again* for the current step or not.

Reply:
Ok. I can add it as an event in the parser.